draft-ietf-ppm-dap.md

```diff
@@ -2595,13 +2511,6 @@ Attacks on other properties of the system:
    existence of a report does not imply the occurrence of a sensitive event.
 1. Aggregators can deny service by refusing to respond to collection requests or
    aggregate share requests.
-1. Collection requests may leak information beyond the collection results. For
```

Comment:
I'm not sure there is a concrete example to cite here anymore, though I suppose there's nothing stopping a singly-collected VDAF with a nontrivial aggregation parameter from leaking information via the aggregation parameter.

Reply:
I think we should leave this in, even without the explicit reference to Poplar1. Even when restricted to collecting batches once, DAP admits many future, unknown VDAFs, and pointing out that VDAFs may leak information (and not just to the collector!) and may need to be composed with DP is a legitimate security consideration. However I don't like the phrasing "Collection requests may leak information beyond the collection results" because it seems tautological. Let's try:

```suggestion
1. Some VDAFs could leak information to either Aggregator or the Collector
   beyond what the protocol intended to learn. It may be possible to mitigate
   such leakages using differential privacy ({{dp}}). 
```